integration-tests/goss/centos7/goss-expected.yaml

```diff
@@ -75,59 +75,12 @@ command:
     - 'sh: foobar: command not found'
     timeout: 10000
 dns:
-  CAA:dnstest.io:
```

Comment:
As above, comment + TODO?

Reply:
Oh - some of these are going to be the golden-reference files, aren't they; those can't have the comments, can they.

Reply:
Yep, comments can't be here.